shakenfist/daemons/triggers.py

```diff
@@ -65,6 +65,7 @@ def run(self):
             all_observers = list(observers.keys())
             for instance_uuid in all_observers:
                 if not observers[instance_uuid].is_alive():
+                    observers[instance_uuid].join(1)
```

Comment:
Maybe a quick comment why join() a not is_alive() process. (Unless everybody knows but me... does happen)

Reply:
Done.